tensorboard/defs/defs.bzl

```diff
@@ -200,3 +200,24 @@ def tf_ng_module(assets = [], **kwargs):
         angular_assets = assets,
         **kwargs
     )
+
+def tf_inline_images(name, html_template, images, out):
+    """Inline png images in html.
+
+    Replaces %*.png% in the input .html file with a data URI containing the
+    base64-encoded image content of the corresopnding .png files in the images/
+    subdirectory.
```

Comment:
We won't be getting an `images` directory per say but rather we would get their file names as input to the rule. So, it would be more correct to say,
```
Replaces %<file_basename>.png% in the input `html_template` with a data URI containing the
base64-encoded image content of the corresopnding .png files in the `images` input.

In case there is a collision in the base file name, the first instance will take precedence over the
others.

Example:
# In html_template
<img src="%my_file.png%" />

# in BUILD
tf_inline_pngs(
  name = "my_rule",
  html_template = "path_to_my_template.html",
  images = [
    "path_to/my_file.png",
  ] + glob("some_folder/*.png"),
  out = "my_filename.html",
)
```

Reply:
Done. Thanks!